python/lsst/sims/catUtils/supernovae/snObject.py

```diff
@@ -589,6 +590,94 @@ def SNObjectSED(self, time, wavelen=None, bandpass=None,
         SEDfromSNcosmo.addCCMDust(a_x=ax, b_x=bx, ebv=self.ebvofMW)
         return SEDfromSNcosmo
 
+    def SNObjectSourceSED(self, time, wavelen=None):
```

Comment:
I think I prefer the name as is. 

`SNObject` is designed to return a lot of things just the way `SNCosmo` would return them. There are a few differences. The SEDS are one of them. Hence, I think it is important to emphasize the difference to SN users who might be more accustomed to the SNCosmo SED. 
Also, `source` is a specific term that is in use. So I think that is useful to keep.